### PR TITLE
add stats handler

### PR DESCRIPTION
**Pull request by anuvedverma**

### CODE REVIEW

**Comment by anuvedverma** [comment]
/ptal @riteshvaryani @theevocater 

Very rough attempt at adding `StatisticsHandler` logs-- lmk if you know a better way to do this!

File: proxyserver/src/main/java/com/lyft/data/proxyserver/ProxyServer.java [hunk 1]
```diff
@@ -142,6 +150,16 @@ public void start() {
 
     try {
       this.server.start();
+
+      // Schedule a task to log metrics at a fixed rate
+      StatisticsHandler stats = this.server.getChildHandlerByClass(StatisticsHandler.class);
+      this.scheduler.scheduleAtFixedRate(() -> {
+        log.debug("(jetty) Num requests: " + stats.getRequests());
+        log.debug("(jetty) Num requests: " + stats.getRequestsActive());
```
Comment (premsantosh) [inline proxyserver/src/main/java/com/lyft/data/proxyserver/ProxyServer.java:158]: Num active requests?

Comment (anuvedverma) [reply proxyserver/src/main/java/com/lyft/data/proxyserver/ProxyServer.java:158]: ah thanks-- fixed it on the `sev-16337-add-header-logging` branch

**Comment by theevocater** [comment]
wait i'm confused -- is the goal here to actually collect these metrics or just log them to stdout? wouldn't it be better to just output this via the statsd collector we already have?

**Comment by anuvedverma** [comment]
@theevocater doing this via statsd collector would probably be much better, but I'm honestly not sure how to do it... I see that we use statsd collector in `prestoproxy`, but don't see it in `presto-gateway`. Or would the idea be to configure the `StatisticsHandler` request logging somewhere in `prestoproxy` instead?

**Pull request merged**